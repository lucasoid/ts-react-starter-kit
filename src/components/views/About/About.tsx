import * as React from 'react';
import { View } from '~components/Views/View';

export const About: React.FC<{}> = () => (
    <View>
        <h2>About</h2>
        <p>
            listshare is an app that helps you stay organized. Make a list, check it twice, and share it with people who
            need it. ðŸŽ…
        </p>
        <p>Built to demonstrate features of React 16.8+ and TypeScript.</p>
    </View>
);
