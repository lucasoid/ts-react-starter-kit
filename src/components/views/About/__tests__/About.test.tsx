import * as React from 'react';
import { shallow } from 'enzyme';
import { About } from '../About';

describe('About', () => {
    it('renders an About page', () => {
        expect(shallow(<About />)).toMatchSnapshot();
    });
});
