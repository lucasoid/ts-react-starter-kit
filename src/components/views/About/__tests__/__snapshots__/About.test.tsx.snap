// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders an About page 1`] = `
<View>
  <h2>
    About
  </h2>
  <p>
    listshare is an app that helps you stay organized. Make a list, check it twice, and share it with people who need it. ðŸŽ…
  </p>
  <p>
    Built to demonstrate features of React 16.8+ and TypeScript.
  </p>
</View>
`;
